specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json

```diff
@@ -6645,6 +6669,15 @@
           },
           "readOnly": true,
           "description": "A list of references to all dedicated hosts in the dedicated host group."
+        },
+        "instanceView": {
+          "$ref": "#/definitions/DedicatedHostGroupInstanceView",
+          "readOnly": true,
+          "description": "The dedicated host group instance view."
+        },
+        "supportAutomaticPlacement": {
+          "type": "boolean",
```

Comment:
we also need to add property "required": false, as Aure picks a default when the value is not provided.

Reply:
I believe "required" in the outer loop with only "platformFaultDomainCount" makes "supportAutomaticPlacement" not a required property.